src/App.tsx

```diff
@@ -43,15 +43,15 @@ function App() {
         },
       ]
 
-      if (fundraisePositionData) {
-        links = [
-          ...links,
-          {
-            title: "CLAIM",
-            route: routes.FundraiseClaim,
-          },
-        ]
-      }
+      // if (fundraisePositionData) {
+      links = [
+        ...links,
+        {
+          title: "CLAIM",
+          route: routes.FundraiseClaim,
+        },
+      ]
+      // }
```

Comment:
should we add this condition back in?

Reply:
Good catch as well 😆 Needed the claim button a bit and forgot about it. Cheers!

Reply:
haha yeah, I figured